[2025-09-10T06:17:00.234552] [INFO] [SYSTEM] Start new run (run_id=626c9061-6ec0-4ab8-bcb3-5567d6cc6e0f)
[2025-09-10T06:17:27.351512] [INFO] [GENERAL] debug_utils.py:log_debug:238 - User int input valid: 3
[2025-09-10T06:17:36.055109] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Counts computed for kit build.
[2025-09-10T06:17:39.714019] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Base64 length bounds computed for pad selection.
[2025-09-10T06:17:44.309486] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Using PAD size (bucketized): 128
[2025-09-10T06:17:47.936898] [INFO] [GENERAL] debug_utils.py:log_debug:238 - User int input valid: 1
[2025-09-10T06:17:49.265801] [INFO] [GENERAL] debug_utils.py:log_debug:238 - User int input valid: 1
[2025-09-10T06:17:50.020194] [INFO] [GENERAL] debug_utils.py:log_debug:238 - User int input valid: 1
[2025-09-10T06:17:50.021194] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Argon2id parameters confirmed for kit.
[2025-09-10T06:17:50.022194] [WARNING] [CRYPTO] debug_utils.py:log_debug:238 - Combinatorial hardness below policy threshold.
[2025-09-10T06:17:52.465300] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Combinatorial hardness evaluation complete.
[2025-09-10T06:17:52.467310] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() via node-bridge
[2025-09-10T06:17:52.766459] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() complete.
[2025-09-10T06:17:52.768459] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() via node-bridge
[2025-09-10T06:17:52.836479] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() complete.
[2025-09-10T06:17:52.837480] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() via node-bridge
[2025-09-10T06:17:53.390375] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() complete.
[2025-09-10T06:17:53.393367] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() via node-bridge
[2025-09-10T06:17:53.946482] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() complete.
[2025-09-10T06:17:55.596067] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Using user-provided password='Rex' (raw).
[2025-09-10T06:17:55.597068] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Starting Argon2id KDF (Noble). pass='Rex', salt(b64)='YA8Fk7l+xjRLPUF9hEwO3A=='
[2025-09-10T06:17:55.723093] [ERROR] [GENERAL] debug_utils.py:log_exception:286 - Error during encrypted kit preparation
